spotify/src/main/java/org/wso2/carbon/connector/spotify.java

```diff
@@ -0,0 +1,36 @@
+/*
+*  Copyright (c) 2005-2010, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*  WSO2 Inc. licenses this file to you under the Apache License,
+*  Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*/
+package org.wso2.carbon.connector;
+
+import org.apache.synapse.MessageContext;
+import org.wso2.carbon.connector.core.*;
+
+public class spotify extends AbstractConnector {
```

Comment:
Is this some class created for testing? .. anyways, never name a Java class starting with a lower case character ..